src/app/app-components/courses/courses-item/courses-item.component-test-host.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { Course } from '../../../../app/app-models/course';
```

Comment:
There's a way to improve the internal dependency paths:

In the `tsconfig.app.json`, add the following under the `compilerOptions:

```json
    "baseUrl": "src/",
    "paths": {
      "@app/*": ["app/*"]
    }
```

Then, you can import like this: 
```typescript
import { Course } from '@app/app-models/course';
```

One other good practice you can introduce, is the usage of barrel files.

Basically, you add a file named `index.ts` and export the all the classes, interfaces, types that you would like.

For example:

```typescript
// on /src/app/app-models/index.ts
export * from './course';
export * from './user';
```

With this, you could import the `Course` interface here like this:

```typescript
import { Course } from '@app/app-models';
```

Feel free to try it out on the next homework.

Reply:
Yes, you are right, I'm not a fan of that kind of imports but I didn't know how to fix it. This is a super awesome pro-tip that I'm going to apply in next PR and in other projects :smiley: 

THANK YOU!!